docs/learn/digital-assets/read-nft-metadata.md

```diff
@@ -0,0 +1,679 @@
+
+<Tabs groupId="provider-lib">
+  <TabItem value="ethers" label="ethers"  attributes={{className: "tab_ethers"}}>
+
+```js
+// Add the necessary imports to your JS file
+import { ethers } from 'ethers';
+import lsp8Artifact from '@lukso/lsp-smart-contracts/artifacts/LSP8IdentifiableDigitalAsset.json';
+
+const provider = new ethers.JsonRpcProvider('https://4201.rpc.thirdweb.com');
+
+// Create contract instance
+const myAssetContract = new ethers.Contract(
+  '<myAssetAddress>',
+  lsp8Artifact.abi,
+  provider,
+);
+```
+
+  </TabItem>
+
+<TabItem value="web3" label="web3" attributes={{className: "tab_web3"}}>
+
+```bash
+npm install web3 @erc725/erc725.js @lukso/lsp-smart-contracts
+```
+
+```js
+// Add the necessary imports to your JS file
+import Web3 from 'web3';
+import lsp8Artifact from '@lukso/lsp-smart-contracts/artifacts/LSP8IdentifiableDigitalAsset.json';
+
+const web3 = new Web3(
+  new Web3.providers.HttpProvider('https://4201.rpc.thirdweb.com'),
+);
+// Create contract instance
+const myAssetContract = new web3.eth.Contract(
+  lsp8Artifact.abi,
+  '<myAssetAddress>',
+);
+```
+
+  </TabItem>
+
+</Tabs>
+
+## Preparing the Token IDs Format
+
+The [LSP8](../../standards/tokens/LSP8-Identifiable-Digital-Asset.md) allows for different [Token ID Formats](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-8-IdentifiableDigitalAsset.md#lsp8tokenidformat), meaning developers can specify their token IDs as `Number`, `String`, `Smart Contract Address`, `Byte Identifier` or `Hash Digest`.
+
+To call the contract, you must first **prepare** your token IDs to match the standardized **Byte32 Hex Strings** based on the [`LSP8TokenIdFormat`](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-8-IdentifiableDigitalAsset.md#lsp8tokenidformat) standardization. The **global token ID format** can be fetched from the [ERC725Y](../../standards/lsp-background/erc725.md#erc725y-generic-data-keyvalue-store) data storage using the [`getData()`](../../contracts/contracts/ERC725/ERC725.md#getdata) function:
+
+<Tabs groupId="provider-lib">
+  <TabItem value="ethers" label="ethers"  attributes={{className: "tab_ethers"}}>
+
+```js
+// ...
+
+// Get the global token ID format
+let tokenIdFormat = parseInt(
+  await myAssetContract.getData(ERC725YDataKeys.LSP8['LSP8TokenIdFormat']),
+  16,
+);
+
+console.log(tokenIdFormat);
+// 0
+```
+
+  </TabItem>
+
+<TabItem value="web3" label="web3" attributes={{className: "tab_web3"}}>
+
+```js
+// ...
+
+// Get the global token ID format
+let tokenIdFormat = parseInt(
+  await myAssetContract.methods
+    .getData(ERC725YDataKeys.LSP8['LSP8TokenIdFormat'])
+    .call(),
+  16,
+);
+
+console.log(tokenIdFormat);
+// 0
+```
+
+  </TabItem>
+
+</Tabs>
+
+Its value can indicate wheater the token ID format is [equal or mixed across all token IDs](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-8-IdentifiableDigitalAsset.md#lsp8tokenidformat)
```

Comment:
typo correction here: 

Its value can indicate whether the token ID format is [equal or mixed across all token IDs](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-8-IdentifiableDigitalAsset.md#lsp8tokenidformat).